internals/scripts/ElectronRebuild.js

```diff
@@ -4,10 +4,10 @@ import fs from 'fs';
 import { dependencies } from '../../app/package.json';
 
 const nodeModulesPath = path.join(__dirname, '..', '..', 'app', 'node_modules');
-
+console.log('Running electron-rebuild');
 if (Object.keys(dependencies || {}).length > 0 && fs.existsSync(nodeModulesPath)) {
   const electronRebuildCmd =
-    '../node_modules/.bin/electron-rebuild --parallel --force --types prod,dev,optional --module-dir .';
+    '../node_modules/.bin/electron-rebuild --parallel --debug --force --types prod,dev,optional --module-dir .';
```

Comment:
Is the `--debug` here committed intentionally?

Reply:
Yeah it doesn't really hurt, shows a slightly more verbose output if there's an error, but good spot